python/lsst/daf/butler/transfers/_yaml.py

```diff
@@ -372,7 +372,7 @@ def load(
         for element, dimensionRecords in self.dimensions.items():
             if skip_dimensions and element in skip_dimensions:
                 continue
-            self.registry.insertDimensionData(element, *dimensionRecords)
+            self.registry.insertDimensionData(element, *dimensionRecords, skip_existing=True)
```

Comment:
Can there be a comment here to say that we are using skip_existing because we are assuming that records that are being imported from an export from another registry are assumed to be trustworthy and not to have been manually altered and we are doing this for speed reasons because sync would be too slow?